Services/Certificate/classes/Helper/ilCertificateDateHelper.php

```diff
@@ -28,12 +28,12 @@ public function formatDate(string $date, $dateFormat = null) : string
     }
 
     /**
-     * @param string $dateTime
-     * @param int $format
+     * @param string|int $dateTime
+     * @param ?int       $dateFormat
      * @return string
      * @throws ilDateTimeException
      */
-    public function formatDateTime(string $dateTime, $dateFormat = null) : string
+    public function formatDateTime($dateTime, ?int $dateFormat = null) : string
```

Comment:
Why did you remove the type here?

Reply:
Because $dateTime can both be a string and an int

Example:
ilDefaultPlaceholderValues::getPlaceholderValues() uses time() => int
ilCoursePlaceholderValues::getPlaceholderValues calls $this->participantsHelper->getDateTimeOfPassed($objId, $userId); which returns a string